content/docs/authorization/_index.md

```diff
@@ -43,7 +43,7 @@ The following diagram shows a high-level overview of CSM for Authorization with
 {{<table "table table-striped table-bordered table-sm">}}
 |               | PowerMax         | PowerFlex | PowerScale |
 |---------------|:----------------:|:-------------------:|:----------------:|
-| Storage Array |5978.479.479, 5978.711.711, Unisphere 9.2|    3.5.x, 3.6.x    | OneFS 8.1, 8.2, 9.0, 9.1, 9.2, 9.3 |
+| Storage Array |5978.479.479, 5978.711.711, Unisphere 10.0|    3.5.x, 3.6.x    | OneFS 8.1, 8.2, 9.0, 9.1, 9.2, 9.3 |
```

Comment:
If we have customers on 9.x, what are we telling them?

Reply:
we should add both 9.x and 10.x

Reply:
We are mandating them to upgrade to 10.x